app/controllers/users_controller.rb

```diff
@@ -0,0 +1,77 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
```

Comment:
- [ ] <b>REQUIRED:</b> It is also the same case here.

Reply:
hey, @rukundoeric ! Thanks for the comments, I made the requested changes. But for this one I don't understand which Association should be used here. Could you elaborate please?